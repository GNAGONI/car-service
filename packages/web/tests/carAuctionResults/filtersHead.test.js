import React from 'react'
import renderer from 'react-test-renderer'
import FiltersHead from '../../components/carAuctionResults/filtersHead'

it('renders correctly', () => {
  const tree = renderer.create(<FiltersHead />).toJSON()
  expect(tree).toMatchSnapshot()
})
