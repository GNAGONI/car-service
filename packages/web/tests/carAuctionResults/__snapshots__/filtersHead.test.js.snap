// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="filters-head"
>
  <div
    className="bar"
  >
    <ul
      className="btn-selection"
    >
      <li
        className=""
        role="presentation"
      >
        <span>
          All listings
        </span>
      </li>
      <li
        className=""
        role="presentation"
      >
        <span>
          Auction
        </span>
      </li>
      <li
        className=""
        role="presentation"
      >
        <span>
          Marked for scrap
        </span>
      </li>
    </ul>
    <div
      className="filters-holder"
    >
      <div
        className="form-group"
      >
        <span
          className="label"
        >
          View
        </span>
        <ul
          className="filter-links"
        >
          <li
            className="active"
            onClick={[Function]}
            role="presentation"
          >
            <a
              className="icon-thumbnail2"
              data-mode="12"
              data-mode-lg="4"
              data-mode-md="6"
            >
              <span
                className="sr-only"
              >
                Thumbnail 9
              </span>
            </a>
          </li>
          <li
            className=""
            onClick={[Function]}
            role="presentation"
          >
            <a
              className="icon-thumbnail"
              data-mode="6"
              data-mode-lg="3"
              data-mode-md="4"
            >
              <span
                className="sr-only"
              >
                Thumbnail 12
              </span>
            </a>
          </li>
        </ul>
      </div>
      <div
        className="form-group"
      >
        <label>
          Sort by:
        </label>
        <div
          className="input-group"
        >
          <div
            className="css-r3335i"
            onKeyDown={[Function]}
          >
            <div
              className="css-55nwcx"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
            >
              <div
                className="css-cys4jg"
              >
                <div
                  className="css-1492t68"
                >
                  
                </div>
                <input
                  className="css-14uuagi"
                  disabled={false}
                  id="react-select-2-input"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  readOnly={true}
                  tabIndex="0"
                  value=""
                />
              </div>
              <div
                className="css-1wy0on6"
              >
                <span
                  className="css-rb20kt"
                />
                <div
                  aria-hidden="true"
                  className="css-1ep9fjw"
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                >
                  <svg
                    aria-hidden="true"
                    className="css-19bqh2r"
                    focusable="false"
                    height={20}
                    viewBox="0 0 20 20"
                    width={20}
                  >
                    <path
                      d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="bar"
  >
    <span
      className="title"
    >
      Showing results 0 - NaN of 0
    </span>
    <div
      className="location"
    >
      <span
        className="subtitle"
      >
        Location:
        <a
          className="opener"
        >
          PR1 1QE
        </a>
      </span>
      <div
        className="location-details "
      >
        <strong
          className="title"
        >
          Update your location
        </strong>
        <div
          className="form-inline"
        >
          <div
            className="form-group"
          >
            <div
              className="input-group"
            >
              <input
                className="form-control"
                placeholder="United Kingdom"
                type="text"
              />
            </div>
          </div>
          <div
            className="form-group"
          >
            <div
              className="input-group"
            >
              <input
                className="form-control"
                placeholder="SK14 R73"
                type="text"
              />
            </div>
          </div>
          <div
            className="btn-block"
          >
            <button
              className="btn btn-primary"
              type="button"
            >
              Update
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
